tests/pytests/unit/modules/test_ps.py

```diff
@@ -260,13 +325,129 @@ def test__get_proc_cmdline():
     ret = ps._get_proc_cmdline(DummyProcess(cmdline=cmdline))
     assert ret == cmdline, ret
 
+    with patch.object(DummyProcess, "cmdline") as mock_cmdline:
+        mock_cmdline.side_effect = psutil.NoSuchProcess(DummyProcess(cmdline=cmdline))
+        ret = ps._get_proc_cmdline(DummyProcess(cmdline=cmdline))
+        assert ret == []
+
+    with patch.object(DummyProcess, "cmdline") as mock_cmdline:
+        mock_cmdline.side_effect = psutil.AccessDenied(DummyProcess(cmdline=cmdline))
+        ret = ps._get_proc_cmdline(DummyProcess(cmdline=cmdline))
+        assert ret == []
+
+
+@pytest.mark.skipif(not ps.PSUTIL2, reason="Only run for psutil 2.x")
```

Comment:
We now require psutil >= 5.0

https://github.com/saltstack/salt/blob/v3006.3/requirements/base.txt#L8

Let's just nuke all of the old code.